dev-python/pbr/files/pbr-5.11.1-importlib-suffixes.patch

```diff
@@ -0,0 +1,34 @@
+Subject: [PATCH] Replace imp with importlib.machinery
+
+The imp was removed in Python 3.12 [1]. This commit replaces
+get_suffixes() method call with _SUFFIXES array from imprtlib.machinery.
+
+[1] https://github.com/python/cpython/issues/98040
+
+Upstream-PR: https://github.com/openstack/pbr/pull/22
```

Comment:
IDK how much Gentoo cares, but PBR doesn't take PRs from Github. However, I did push this to the proper upstream place: https://review.opendev.org/c/openstack/pbr/+/884789 -- you may want to update this link to point there instead (alternatively, people can still find it, because I linked it in the PR)

Reply:
Thanks, I will update the link.